File: PRG_L1_PY/06 [2] 2016년.py
def solution(a, b):
    d = [31,29,31,30,31,30,31,31,30,31,30,31]
    for i in range(a-1):
        b += d[i]
    print(b)
    answer = ["FRI","SAT","SUN","MON","TUE","WED","THU"][(b-1)%7]
    return answer

print(solution(5,24))

File: L1/PRG_L1_C/06 [2] 2016년.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(int a, int b) {
	int day[] = { 31,29,31,30,31,30,31,31,30,31,30,31 };
	char *name[] = { "THU","FRI","SAT","SUN","MON","TUE","WED" };
	for (int i = 0; i < a-1; i++)
		b += day[i];
	char* answer = (char*)malloc(sizeof(char) * 3);
	answer = *(name + b % 7);
	return answer;
}

int main() {
	printf("%c", *solution(5, 24));
}